src-tauri/src/system_tray.rs

```diff
@@ -1,8 +1,8 @@
 use std::thread;
 use std::time::Duration;
 
-use nix::sys::signal::{self, Signal};
-use nix::unistd::Pid;
+use tauri::api::process::CommandChild;
```

Comment:
This is not used. is the holochain process shutdown correctly without kill child explicitly?

Reply:
on Windows and Linux, I didn't have any issues not sure about macOS.